### ISSUE TITLE
Kibana server not available (kibana_settings container waiting forever)

### ISSUE THREAD

**Issue by dfmcub** [elastic/start-local#19]
When I run the `start-local` script with curl, I can access the ElasticSearch database ( `es-local-dev` container ) **but not the Kibana server** ( `kibana-local-dev` container ). 
I believe the main reason is that `kibana_settings` container never completes [the creation of user kibana_system in the elasticsearch database ](https://github.com/elastic/start-local/blob/c299c58ef308fba82ce9052d188304c41413d8c6/start-local.sh#L521).

Additionally, the logs at `es-local-dev` container continously report that the servers [es-container-logs.log](https://github.com/user-attachments/files/17663808/es-container-logs.log)

**Comment by ezimuel** [comment]
@dfmcub This issue is related to the high disk watermark [90%] exceeded on Elasticsearch (default value). We need to adjust this limit without using the percentage, e.g. requiring at least 4 GB of disk space. I'm working on a fix.

**Comment by dfmcub** [comment]
Hi @ezimuel ,

Thanks for the answer :D . I just saw that, strangely, the problem has a chance to solve itself if I manually purge the elastic and kibana images and then running the start_local script again. There's no guarantee it will work on the first try, though.

**Comment by ezimuel** [comment]
Yes, I'm adding a check in the start-local itself as first requirement. Moreover, I'm changing the Elasticsearch settings to require only 2 GB (and not 10% of the disk space).

**Comment by ezimuel** [comment]
@dfmcub can you try to execute the [start-local.sh](https://github.com/elastic/start-local/blob/main/start-local.sh) in main? This should fix the 90% limit disk space. Thanks!

**Comment by dfmcub** [comment]
Hi @ezimuel 
I just tested the script in the following circumstances, since I saw that the Pull request hasn't been applied yet:

1. Download the [script at main branch](https://github.com/elastic/start-local/blob/main/start-local.sh) and running it normally from terminal : ` $ ./start-local.sh `
Result: **It works perfectly**   ✅ !!  You can see the logs in [this zip file I include ](https://github.com/user-attachments/files/17734448/es_start_local_logs.zip)
2. Clone the [fix/19](https://github.com/elastic/start-local/tree/fix/19) branch where your [pull request](https://github.com/elastic/start-local/pull/22) is and run the bashunit tests
Result: **All tests passed**   ✅ !! 

Many thanks for the hard work 💯  👍 !!

**Comment by ezimuel** [comment]
Thanks @dfmcub for testing. I'm going to release a new version of start-local soon.
